node.js

```diff
@@ -49,7 +49,42 @@ module.exports = {
     'space-before-function-paren': off,
     'keyword-spacing': error,
     'eol-last': [ error, always ],
-    'brace-style': [ error, '1tbs', { allowSingleLine: true } ],
-    'curly': [ error, all ]
+    'brace-style': [ error, '1tbs', { allowSingleLine: false } ],
+    'curly': [ error, all ],
+    'block-spacing': [ error,  always ],
+    'indent': [ error, 2, { SwitchCase: 1 } ],
```

Comment:
Enforce 2 spaces, and make `switch/case` statements indented by 2 spaces too
https://eslint.org/docs/rules/indent
```js
// Old
switch (something) {
case 1:
          thing()
  case 2:
something()
}
// New
switch (something) {
  case 1:
    thing()
  case 2:
    something()
}
```

Reply:
So, I guess, there is no way to convince anyone to go to 4 spaces, right?